src/main/java/ch/wisv/toornament/ToornamentClient.java

```diff
@@ -1,86 +1,89 @@
-package ch.wisv.toornament;
-
-import com.fasterxml.jackson.databind.ObjectMapper;
-
-import java.io.IOException;
-import java.text.SimpleDateFormat;
-
-import ch.wisv.toornament.concepts.Tournaments;
-import ch.wisv.toornament.model.request.ApiTokenRequest;
-import ch.wisv.toornament.model.response.ApiTokenResponse;
-import okhttp3.MediaType;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.RequestBody;
-import okhttp3.Response;
-
-public class ToornamentClient {
-    public static final MediaType JSON
-        = MediaType.parse("application/json; charset=utf-8");
-    private OkHttpClient httpClient;
-    private String apiKey;
-    private String clientId;
-    private String clientSecret;
-    private String oAuthToken;
-    private ObjectMapper mapper;
-    SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Then we can just add that line back in. This is a lot of jumping through hoops for something very minor.